karma.conf.js

```diff
@@ -97,15 +97,15 @@ module.exports = function (config) {
           },
           // NOTE: For better debugging you can comment out the
           // istanbul-instrumenter-loader below
-          {
```

Comment:
Should this be commented out - seems like this is a change to the coverage information we are seeing.

Reply:
Karma hates the comlink package. `coverageIstanbulReporter` is responsible for coverage based on my understanding not this one